logs-aggregation/README.md

```diff
@@ -0,0 +1,81 @@
+# Logs aggregations
+
+We (Packit) want to have access to logs from our worker pods even after
+the pods are restarted (due to image or cluster update).
+And ideally, logs from all workers aggregated in one place so that one does
+not have to grep several logs. We don’t need it for staging,
+but would rather test it on staging first.
+
+## Options
+
+### Testing Farm’s Loki
+
+Initially, we wanted to use [TestingFarm's Loki](http://loki.tft.osci.redhat.com),
+but we realized it's in VPN and hence can't be reached by a [Promtail](https://grafana.com/docs/loki/latest/clients/promtail/)
+sidecar container running in the Automotive cluster(s) outside the VPN.
+
+### CKI’s Loki in Managed Platform
+
+CKI has a Loki instance as well, but it’s also in VPN.
+
+```text
+Michael Hofmann, Jun 3rd, 10:22 AM
+We've been thinking of moving it to MP+ where you can have external access to it as well, but this is on our backlog
+```
+
+### Managed Loki on Grafana cloud
+
+[Free forever access](https://go2.grafana.com/loki-grafana-cloud.html):
+50GB of logs, 14-day retention for logs, access for up to 3 (!) team members
```

Comment:
oh

well we could probably pay for it but since we have splunk... we should use that one if possible